src/DDTrace/Integrations/PDO/PDOIntegration.php

```diff
@@ -394,15 +405,15 @@ public static function storeStatementFromConnection($pdo, $stmt)
             // When an error occurs 'FALSE' will be returned in place of the statement.
             return;
         }
-        $pdoHash = spl_object_hash($pdo);
+        $pdoHash = is_object($pdo) ? spl_object_hash($pdo) : '';
```

Comment:
The pattern `is_object($x) ? spl_object_hash($x) : ''` is used in at least 3 places; should we make this a helper method?

Reply:
Good point. I'd prefer to keep it as-is since we're already making quite a few static method calls in this integration. I also don't want to make too many changes to the existing integration. But I can totally be convinced if you really think it needs it. :)